salt/transport/tcp.py

```diff
@@ -432,7 +432,15 @@ def socket(self):
 
     def close(self):
         if self._socket is not None:
-            self._socket.shutdown(socket.SHUT_RDWR)
+            try:
+                self._socket.shutdown(socket.SHUT_RDWR)
+            except socket.error as exc:
+                if socket.errno.ENOTCONN == 107:
```

Comment:
I am a bit confused as to what this line does:

```
if socket.errno.ENOTCONN == 107:
```

Isn't `socket.errno.ENOTCONN` a constant? If so, the condition states that if the constant value is 107, then do nothing. This doesn't make much sense to me. I would expect comparing some value within the exception `exc` instead.